packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -215,3 +216,16 @@ const isFilteredDataProvider = (dataProvider: IPresentationTreeDataProvider | IF
 const getFilteredDataProvider = (dataProvider: IPresentationTreeDataProvider | IFilteredPresentationTreeDataProvider): IFilteredPresentationTreeDataProvider | undefined => {
   return isFilteredDataProvider(dataProvider) ? dataProvider : undefined;
 };
+
+const useModelsTreeRenderer = () => {
```

Comment:
```suggestion
function getModelsTreeRenderer() {
```

This function does not use any hooks so there is no reason to add `use` to the name